src/snowflake/cli/plugins/connection/commands.py

```diff
@@ -55,7 +64,7 @@ def _mask_password(connection_params: dict):
     return connection_params
 
 
-@app.command(name="list")
+@app_creator.command(name="list")
```

Comment:
Using `app` will reduce a lot of changes, I don't think we need to to the rename

Reply:
changed